mne/_fiff/compensator.py

```diff
@@ -42,11 +42,9 @@ def _make_compensator(info, grade):
             for col, col_name in enumerate(this_data["col_names"]):
                 ind = [k for k, ch in enumerate(info["ch_names"]) if ch == col_name]
                 if len(ind) == 0:
-                    raise ValueError(
-                        "Channel %s is not available in " "data" % col_name
-                    )
+                    raise ValueError(f"Channel {col_name} is not available in " "data")
```

Comment:
`ruff` has added this `"blah" "blah"` pattern a bunch of places. In a follow-up PR I'll use `git grep "^.*\".*\" \".*\".*$"` to find them and manually remove them. But don't want to do it in this PR because the diff is already annoying large. Both PRs should end up in our `git-blame-revs-ignore` anyway